### DISCUSSION TITLE
How to achieve image consistency with 2 identic prompts (same seed) having same 3D model viewed from 2 different angles

### DISCUSSION THREAD

**Discussion by hypergeff** [lllyasviel/ControlNet#551] [Q&A] [answered]
Hello team,

I have a 3D model from which I generate 2 sets of lineart, depth and normal maps viewed from 2 different 3D view angles.
Then I generate 2 identic prompts (same seed and everything). The 2 result images are pretty similar but some part colors applied on the 3D model are different at some locations between the 2 images (for instance the roof of a vehicle is black on first image and yellow on second image).
Are there some parameters or something to fine tune in order to guaranty the exact same 3D model coloring between the 2 images?

Thanks!

![image (30)](https://github.com/lllyasviel/ControlNet/assets/5797578/7798def0-9565-4b67-ab0c-e0d545ed177e)
![image (31)](https://github.com/lllyasviel/ControlNet/assets/5797578/e1647177-608a-4acc-92a0-f848947c1b4b)

**Comment by geroldmeisinger** [comment, marked as answer]
No. This is fundamentally not how Stable Diffusion works which generates a new image everytime. Having said that, there is a lot of on-going research and community effort to "simulate" exactly what you want to achieve. You need to do some research. Look at all the video2video conversions of animate girls dancing which is basically the same as your excavator. Some flicker more, some flicker less.
What you can try is:
* CharTurner!
* Color control net
* Train your images into a Lora (dreambooth)
* Look into NERF technology: https://github.com/threestudio-project/threestudio
* all the video technologies (animatediff, pika labs, temporal net, and all video2video workflows and tutorials)
* if you feel adventurous, guassian splatting is the hot shit
* manual frame by frame corrections

Don't expect anything to fix your problem though. Good luck!

**Comment by hypergeff** [comment]
Thank you @geroldmeisinger! That's helpful
